src/utils/formValidationUtils.js

```diff
@@ -0,0 +1,16 @@
+export const validateNumberWithinRange = ({
```

Comment:
You can merge the logic of `validateNumberMin` into `validateNumberWithinRange`. Find a way of doing this